sympy/core/function.py

```diff
@@ -1055,6 +1055,11 @@ def _diff_wrt(self):
         else:
             return False
 
+    @property
+    def is_real(self):
+        if self.expr.is_real and self.expr.is_Function and self.expr.args[0].is_real:
+            return True
```

Comment:
What if one of them is `False`, the whole expression should return `False`. If all of them are `None` then it should return `None`. Isn't it?